handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -902,7 +898,8 @@ public final SSLEngineResult unwrap(
                                 if (!dst.hasRemaining()) {
                                     // Move to the next dst buffer as this one is full.
                                     continue;
-                                } else if (packetLength == 0) {
+                                }
+                                if (packetLength == 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 5 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS134)